docs: Update use description of travis_test container

okurz opened the pull request.

kalikiana commented:
Is this an alternative to #2587 or vice versa? A description might help answer that

okurz commented:
@kalikiana wasn't an alternative, more like a subset. I updated the description and removed another commit. I assume that the changes left in this PR are now trivial and also pass tests.
